src/adapter/src/catalog.rs

```diff
@@ -952,7 +979,7 @@ pub struct CatalogEntry {
 pub enum CatalogItem {
     Table(Table),
     Source(Source),
-    Log(&'static BuiltinLog),
+    Log(BuiltinLog),
```

Comment:
This will change even more, as we implement the cleanup. Probably the member has to keep track of which cluster/replica id it belongs to etc.

I was also thinking of a better name, and I think `RecordedLog` would match nicely the `RecordedView` terminology.